domain/src/unit-test/java/uk/gov/moj/sdt/domain/visitor/AbstractDomainObjectVisitorTest.java

```diff
@@ -0,0 +1,83 @@
+package uk.gov.moj.sdt.domain.visitor;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import uk.gov.moj.sdt.domain.api.*;
+import uk.gov.moj.sdt.utils.visitor.Tree;
+
+import static org.junit.Assert.assertThrows;
+
+class AbstractDomainObjectVisitorTest {
```

Comment:
Make the class extend the AbstractSdtUnitTestBase class.  You'll need to add the Override annotation to setUp() method, or you could rename it to setUpLocalTests(), add the Overrider annotation and remove the BeforeEach annotation.